### ISSUE TITLE
xs:annotations not support

### ISSUE THREAD

**Issue by ghilainm** [andrewbober/xsd2jsonschema#10]
The code seems to not support xs:annotations.

Here is an example of failing xsd:
```xsd
<?xml version="1.0" encoding="UTF-8"?>
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" >
  <xs:simpleType name="XXXX">
    <xs:annotation>
      <xs:documentation>
        <usedBy>Used somewhere over a rainbow</usedBy>
      </xs:documentation>
    </xs:annotation>
    <xs:restriction base="xs:token">
      <xs:pattern value=".{1,24}" />
    </xs:restriction>
  </xs:simpleType>
</xs:schema>
```

Remove the xs:annotations and it works fine :)

**Comment by andrewbober** [comment]
Hi @ghilainm  - Thanks for the feedback.  I can see what's wrong.  It's actually the xs:documentation processing.  I have a fix I will publish tomorrow after I complete unit tests.

**Comment by ghilainm** [comment]
@andrewbober  Thx a lot! By the way how do you activate debug of the tool?

**Comment by andrewbober** [comment]
Xsd2JsonSchema uses the [debug](https://www.npmjs.com/package/debug) library for logging.  Enable debugging output by setting the DEBUG environment variable.

I like to use this setting:
export DEBUG=xsd2jsonschema:BaseConverter,xsd2jsonschema:ParsingState

For all possible logs use this setting (warning - this results in a ton of logging)
export DEBUG=xsd2jsonschema:*

**Comment by andrewbober** [comment]
This is resolved in version 0.3.2
